### ISSUE TITLE
[Feature request] Add media from browser

### ISSUE THREAD

**Issue by MiMoHo** [osmandapp/Osmand#7128]
I wish to manually add media from the gallery. But at this time, media can only be created and added at the very moment as there is no option to choose media to insert into AVNotes from a file browser.

**Comment by kufue** [comment]
I agree, this is one of the first things I noticed, I now have to install an app that redirects camera request to gallery, and I didn't find a free licenced one, so I think this feature has to be added soon

**Comment by vshcherb** [comment]
You can simply "rename" files and put it in av-notes, algorithm to rename is pretty simple https://wiki.openstreetmap.org/wiki/Shortlink
